src/main/resources/templates/fragments/user-header.html

```diff
@@ -11,8 +11,7 @@
             </div>
             <div class="header-proposition">
                 <div class="content">
-                    <a href="#proposition-links" class="js-header-toggle menu">Menu</a>
-                    <nav lang="en" th:replace="fragments/user-nav :: nav"> </nav>
+                    <p class="sign-in">Sign in</p>
```

Comment:
you have completely removed the navigation?

Reply:
Yes. There is no need for any navigation on this page. Once successfully logged in the user will be redirected to their domain with full navigation options. The menu was completely empty anyway.